ArduPlane/mode_rtl.cpp

```diff
@@ -18,6 +18,13 @@ bool ModeRTL::_enter()
             return true;
         }
 
+        // treat RTL as QLAND if we are armed but not flying
+        // (when disarmed, allow RTL mode, so pilots can test switches)
+        if (plane.arming.is_armed() && !plane.is_flying()) {
```

Comment:
to make this tighter, it could be in a Q mode, and throttle is idle?

Reply:
ping @robertlong13